README.md

```diff
@@ -1 +1,31 @@
-# protofolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO MICROVERSE
+
+> AN HTML AND CSS FILE.
+
+![screenshot](./IMG1.png)
+
```

Comment:
## [OPTIONAL]

- [ ] to make your `README` look more professional I suggest using a valid screenshot of the project. The current one is broken.